rpcs3/Emu/Audio/Null/NullAudioBackend.h

```diff
@@ -4,21 +4,25 @@
 
 class NullAudioBackend : public AudioBackend
 {
+private:
```

Comment:
I'd rather have the private stuff at the bottom, because you are usually more interested in the public interface when you look at headers